### PR TITLE
Dl 71 grid averaging and 10min windspeeds

### PR DESCRIPTION
**Pull request by nicreask**

### CODE REVIEW

**Review by mizuguta — APPROVED** [review]
LGTM

File: tools/get_tcwind_csv.py [hunk 1]
```diff
@@ -68,26 +100,113 @@ def get_hazard(all_lats, all_lons, terrain_correction,
...
+        min_lon, min_lat, max_lon, max_lat = test_cell.bounds
+        assert max_lat - min_lat == regrid_res*RKG_RES, "Error in regridding, wrong dlat."
+        assert max_lon - min_lon == regrid_res*RKG_RES, "Error in regridding, wrong dlon."
+
+        df = pd.concat(dfs)
+
+        # Get mean of wind speed across locations and replace center cell wind speed
+        if regrid_op == 'mean':
+            df_ws = pd.concat(dfs_ws).groupby('location_name').mean()
+        elif regird_op == 'median':
```
Comment (jasonre0) [inline tools/get_tcwind_csv.py:183]: regird - ? typo ?

**Review by jasonre0 — CHANGES_REQUESTED** [review]
lgtm except for the regird defect.

**Comment by mizuguta** [comment]
Unit-tests failed due to 400 HTTP error. Does it mean we need to update api-server as well?

**Review by mizuguta — APPROVED** [review]
LGTM.

**Pull request merged**